seqr/views/apis/data_manager_api_tests.py

```diff
@@ -819,194 +828,194 @@ def _check_rna_sample_model(self, individual_id, data_source, tissue_type):
     @mock.patch('seqr.views.apis.data_manager_api.load_uploaded_file')
     @mock.patch('seqr.utils.file_utils.subprocess.Popen')
     @mock.patch('seqr.views.apis.data_manager_api.gzip.open')
-    def test_update_rna_seq(self, mock_open, mock_subprocess, mock_load_uploaded_file,
+    def _test_update_rna_seq(self, data_type, mock_open, mock_subprocess, mock_load_uploaded_file,
                             mock_os, mock_datetime, mock_send_slack):
         url = reverse(update_rna_seq)
         self.check_data_manager_login(url)
 
-        for data_type, params in self.RNA_DATA_TYPE_PARAMS.items():
-            with self.subTest(data_type):
```

Comment:
this needed to be broken into separate tests instead of a subTest because the different dat atypes share Sample models now and modify them so the tests have unexpected interactions, but if they are broken into explicit separate tests they each run on a clean database and work as expected

Reply:
The only change from here down is indentation, nothing was actually changed